client/src/fixtures/dataset.ts

```diff
@@ -0,0 +1,35 @@
+import { UPDATE_FREQUENCY } from "src/constants";
+import type { Dataset, DatasetFormData } from "src/definitions/datasets";
+
+export const getFakeDataSet = (dataSet: Partial<Dataset>): Dataset => {
```

Comment:
(Nit) Ailleurs dans la codebase on traite "dataset" comme un nom entier ("dataset" plutôt que "data set", comme en anglais), cf le nom du type `Dataset`

Par ailleurs on pourrait aussi permettre `getFakeDataset()` ? (Actuellement il faudrait écrire `getFakeDataset({})`)

```suggestion
export const getFakeDataset = (dataset: Partial<Dataset> = {}): Dataset => {
```